CalculatorTest.py

```diff
@@ -20,6 +20,9 @@ def test_run_operation_divide(self):
         with self.assertRaises(ZeroDivisionError):
             Calculator.run_operation(9, 0, '/')
 
+    def test_run_operation_modulo(self):
```

Comment:
Great test case! You should check for what would happen if the user does not input anything for the operation

Reply:
good suggestion, how does the program handles empty inputs? if u were to implement this you should also find a way for the user to restart the operation instead of running the program again

Reply:
Great test case